examples/gke/gke.sh

```diff
@@ -204,9 +194,20 @@ kubectl -n cert-manager rollout status --timeout=5m deployment.apps/cert-manager
 echo "Starting the scylla operator..."
 kubectl apply -f ../common/operator.yaml
 
+kubectl wait --for condition=established crd/nodeconfigs.scylla.scylladb.com
 kubectl wait --for condition=established crd/scyllaclusters.scylla.scylladb.com
 wait-for-object-creation scylla-operator deployment.apps/scylla-operator
 kubectl -n scylla-operator rollout status --timeout=5m deployment.apps/scylla-operator
+kubectl -n scylla-operator rollout status --timeout=5m deployment.apps/webhook-server
+
+echo "Configuring nodes..."
+kubectl apply -f nodeconfig-alpha.yaml
+wait-for-object-creation default nodeconfig.scylla.scylladb.com/cluster
```

Comment:
isn't this what `kubectl apply` has already done?

Reply:
I thought it only means that the request was processed by the server? Even the docs explicitly mention waiting for the endpoint to be created https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition.

Reply:
this is a CR, not a CRD. when a CR or any object like Pod is created, it is committed to etcd.
(CRD registration is asynchronous and waits for with `kubectl wait --for condition=established`. `wait-for-object-creation` is usually used to check objects created by other controllers / asynchronously.)

Reply:
Sorry, I misread and thought it referred to `kubectl wait --for condition=established crd/nodeconfigs.scylla.scylladb.com` a couple lines above. Removed.